bin/ci2gubernator.sh

```diff
@@ -29,6 +29,11 @@ openssl aes-256-cbc -d -in "${ENCRYPTED_SA_JSON}" -out "${TMP_SA_JSON}" -k "${GC
 
 go get -u istio.io/test-infra/toolbox/ci2gubernator
 
+if [[ -d "/go/bin/ci2gubernator" ]];then
```

Comment:
not sure why /go/bin/cigubernator would be the target of a go get operation.  Entirely depends on environment variables.

Reply:
ok,i go to fix it.